bin/inx-chronicle/src/api/routes.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use axum::{handler::Handler, routing::get, Extension, Router};
+use chronicle::db::{
+    model::sync::{SyncData, SyncRecord},
+    MongoDatabase,
+};
+use futures::TryStreamExt;
+use hyper::Method;
+use mongodb::{bson::doc, options::FindOptions, Database};
+use tower_http::{
+    catch_panic::CatchPanicLayer,
+    cors::{Any, CorsLayer},
+    trace::TraceLayer,
+};
+
+use super::{error::APIError, responses::*, APIResult};
+
+pub fn routes(db: MongoDatabase) -> Router {
+    #[allow(unused_mut)]
+    let mut router = Router::new().route("/info", get(info)).route("/sync", get(sync));
+
+    #[cfg(feature = "api-v2")]
+    {
+        router = router.nest("/v2", crate::api::v2::routes())
+    }
+
+    #[cfg(feature = "api-metrics")]
+    {
+        router = router.merge(crate::api::metrics::routes());
+        router = router.layer(crate::api::metrics::MetricsLayer);
+    }
+
+    Router::new()
+        .nest("/api", router)
+        .fallback(not_found.into_service())
+        .layer(Extension(db))
+        .layer(CatchPanicLayer::new())
+        .layer(TraceLayer::new_for_http())
+        .layer(
+            CorsLayer::new()
+                .allow_origin(Any)
+                .allow_methods(vec![Method::GET, Method::OPTIONS])
+                .allow_headers(Any)
+                .allow_credentials(true),
+        )
+}
+
+async fn info() -> InfoResponse {
+    let version = std::env!("CARGO_PKG_VERSION").to_string();
+    let is_healthy = true;
+    InfoResponse {
+        name: "Chronicle".into(),
+        version,
+        is_healthy,
+    }
+}
+
+async fn sync(database: Extension<Database>) -> APIResult<SyncDataResponse> {
+    let mut res = database
+        .collection::<SyncRecord>("sync")
```

Comment:
Should this maybe be part of the `Model` trait too? Also, we would need to differentiate between `stardust_sync` and something like `chrysalis_sync` in the future, right?

Reply:
We should not need to differentiate them, since this just keeps track of milestone indexes.